package com.itshare.exceptions.example1;

public class Test {
	
	public static void main(String[] args) {
		
		Test x ;
		
		int num1 = 10;
		int num2 =0;
		
		try {
			
			System.out.println(num1/num2);
			
			
		}catch (ArithmeticException e) {

			 num2 =1;
			System.out.println("Exception :   "+num1/num2);

		}
		
		
		
		
		
		
		//System.out.println(num1/num2);

		
		
	}

}
